tests/vmi_multus_test.go

```diff
@@ -1084,6 +1110,16 @@ func runSafeCommand(vmi *v1.VirtualMachineInstance, command string) error {
 	}, 15)
 }
 
+func getSRIOVInterfaceNameByMAC(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance, mac string) (string, error) {
```

Comment:
Please drop SRIOV from the name.

Also in the error text below.

Reply:
Done.